.github/workflows/update-config.yml

```diff
@@ -0,0 +1,97 @@
+name: Pull Request Action
+permissions: write-all
+on:
+  workflow_dispatch:
+
+env:
+  GH_TOKEN: ${{ secrets.UPDATE_CONFIG_PAT }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Install jq and yq
+        run: |
+          sudo apt-get -y install jq
+          sudo wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq
+          sudo chmod +x /usr/bin/yq
+      - name: Get UbiquiBot Token
+        uses: tibdex/github-app-token@v1.7.0
+        id: get_installation_token
+        with:
+          app_id: ${{ secrets.UBIQUITY_BOUNTY_BOT_APP_ID }}
+          private_key: ${{ secrets.UBIQUITY_BOUNTY_BOT_PRIVATE_KEY }}
+
+      - name: Update Config Params and Create Pull Requests
+        run: |
+         urls="https://github.com/EtherealGlow/ubiquibot-pr-qa"
+         
+         for url in $urls
+          do
+            repoName=$(basename $url)
+            ownerName=$(echo $url | awk -F/ '{print $(NF-1)}')
+            
+            git clone $url $repoName
+            cd $repoName
+            defaultBranch=$(git branch --show-current)
+            
+            # make a branch to update config #
+            git branch update
+            git checkout update
+            
+            curl -sSL https://raw.githubusercontent.com/ubiquity/ubiquibot/development/ubiquibot-config-default.json > default.json
+            declare -A param_mapping=(
+              ["evm-network-id"]="network-id chain-id"
+              ["price-multiplier"]="base-multiplier"
```

Comment:
Whenever we have different param names, we should update hardcoded values, right? 

What is the benefit of this workflow rather than updating manually?  @rndquu

Reply:
> What is the benefit of this workflow rather than updating manually? @rndquu

What if we have 100 partners [here](https://github.com/ubiquity/devpool-directory/blob/development/projects.json) and there is a refactor from `price-multiplier` to `base-price-multiplier`?

I understand that we could set the new param in the default config but I really prefer things to be explicit. Besides the default config value of the new param may differ from the one set in the obsolete partner's config.

Reply:
makes sense